e2e/pages/events/uploadC2DocumentsEvent.page.js

```diff
@@ -24,6 +35,20 @@ module.exports = {
     I.fillField(this.fields.description, description);
   },
 
+  async uploadC2SupportingDocument(file) {
+
+    await I.addAnotherElementToCollection();
```

Comment:
Should be part of the test rather than the page in my opinion.

Reply:
i think i agree with you. I just followed same approach as other tests but happy to move all... what do you think?

Reply:
Don't have opinion here - whatever works better for you.

Reply:
I am going to leave this for now, i think!